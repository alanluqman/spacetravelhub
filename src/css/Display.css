.container {
  max-width: 1000px;
  margin: auto;
  margin-top: 30px;
  display: flex;
  flex-direction: column;
  gap: 20px;
}

.display-section {
  width: 100%;
}

table,
tr {
  width: 100%;
}

table {
  margin-top: 15px;
}

@media screen and (min-width: 500px) {
  .container {
    flex-direction: row;
  }
}
