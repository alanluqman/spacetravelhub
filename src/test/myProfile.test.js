import {
  render, screen, act,
} from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Display from '../pages/MyProfile';

describe('My Profile component', () => {
  afterEach(() => {
    act(() => store.dispatch({
      type: 'spacetravelhub/missions/fetch',
      payload: [],
    }));
  });
  // check for My Missions and My Rockets availability
  it('Check My Missions and My Rockets are available', async () => {
    render(
      <Provider store={store}>
        <Display />
      </Provider>,
    );
    expect(screen.getByText('My Missions')).toMatchSnapshot();
    expect(screen.getByText('My Rockets')).toMatchSnapshot();
  });
});
