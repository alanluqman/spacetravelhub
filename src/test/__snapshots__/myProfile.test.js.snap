// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Profile component Check My Missions and My Rockets are available 1`] = `
<h1>
  My Missions
</h1>
`;

exports[`My Profile component Check My Missions and My Rockets are available 2`] = `
<h1>
  My Rockets
</h1>
`;
